gdx/src/com/badlogic/gdx/scenes/scene2d/Group.java

```diff
@@ -33,6 +34,15 @@
  * @author mzechner
  * @author Nathan Sweet */
 public class Group extends Actor implements Cullable {
+
+	/** Determines how to choose the child that has been hit in a group. */
+	public enum HitMode {
+		/** Return the first child in the hierarchy that has been hit. */
+		first,
```

Comment:
Libgdx's standard with Enums is camel case (not sure why we don't follow the standard and use all caps).